python/neuroglancer/pipeline/volumes/volumes.py

```diff
@@ -10,10 +10,13 @@
 from tqdm import tqdm
 from PIL import Image
 
-from neuroglancer.lib import Vec, clamp, mkdir, COMMON_STAGING_DIR, xyzrange
+from neuroglancer.lib import Vec, eclamp, clamp, mkdir, COMMON_STAGING_DIR, xyzrange
 
-def generate_slices(slices, maxsize):
+def generate_slices(slices, minsize, maxsize, bounded=True):
   """Assisting function for __getitem__. e.g. vol[:,:,:,:]"""
+
+  clampfn = (lambda v,x,y: v) if not bounded else eclamp
```

Comment:
fwiw, originally the design was clamped, but as we made it so you had to be more precise, i swapped clamp for eclamp